internal/route/parser.go

```diff
@@ -47,13 +47,12 @@ func NewParser() (*Parser, error) {
 				{Name: "BindParameterEnd", Pattern: `[},]`, Action: stateful.Pop()},
 			},
 			"BindParameterValue": {
-				stateful.Include("Common"),
```

Comment:
Once entered "BindParameterValue" state, the only possible thing to read it a regex surrounded by `/`s. Thus removing rules from "Common" state.